### DISCUSSION TITLE
Custom translators

### DISCUSSION THREAD

**Discussion by Liametc** [Autodesk/maya-usd#1391] [Q&A]
Does the documentation for AL's custom translators work for the adsk plugin? For instance saving a custom maya node and recalling it with the `mayaUsdImport/Export` commands

Its just a little unclear

Thanks

**Comment by kxl-adsk** [comment]
Sorry for late response. AL plugin uses a different translation framework than the core one. Pixar and Adsk plugins share the same translation framework and translators. This is as well what mayaUsdImport/Export commands will call into.

So no, documentation for AL custom translators will not work. The best place to start exploring translators is https://github.com/Autodesk/maya-usd/tree/dev/lib/usd/translators.

I'm not aware of good documentation explaining how to write custom translators - something to improve for sure. In the mean time, don't hesitate to ask questions.

Cheers,
KxL
